azext_iot/common/_azure.py

```diff
@@ -12,64 +12,71 @@
 
 
 def _get_aad_token(cmd, resource=None):
-    '''
+    """
```

Comment:
Can you please itemize what was changed in this module? Or are these all formatting updates.